src/main.rs

```diff
@@ -9,18 +9,28 @@
 extern crate diesel;
 #[macro_use]
 extern crate rocket;
+#[macro_use]
+extern crate tracing;
 
 mod application;
 mod errors;
 mod routes;
 mod storage;
 mod web;
 
+use tracing_subscriber::{fmt, EnvFilter};
+
 fn main() {
+    // set up logging of applications events to stdout
```

Comment:
Shouldn't logs go to stderr?

Reply:
Yeah, I'm not sure why stdout is the default option here. I'll see if we can change that.